packages/core/package.json

```diff
@@ -46,6 +46,6 @@
     "eslint": "^8.39.0",
     "tsup": "^6.7.0",
     "typescript": "^5.0.4",
-    "zod": "^3.20.2"
+    "zod": "^3.21.0"
```

Comment:
Revert all of these. I dont want to bump this over the peerDep limit so we dont accidentally use new fancy festures